.circleci/config.yml

```diff
@@ -39,6 +39,14 @@ jobs:
           at: "."
 
       - run: npm run test
+      - store_test_results:
+          path: ./reports/junit
+      - store_artifacts:
+          path: ./reports/junit
```

Comment:
Circle requires these steps so that we get nicer summaries of test results in the `Tests` tab and save the artifacts from the test run (screenshots, xml files etc.) in the `Artifacts` tab.